### PR TITLE
T4 pb 12150

### PR DESCRIPTION
**Pull request by nobuotakamasa**
### Description

adding ECU CMOS Battery condition monitoring node.
if codition variable CMAKE_ECU_PLATFORM is "adlink",  this node is enable.
Otherwise it will be disabled.

### Pre-review checklist for the PR author

The PR author must check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
- [ ] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).

### In-review checklist for the PR reviewers

- [ ] The PR reviewers must check the checkboxes below before approval.
- [ ] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).
- [ ] The PR has been properly tested.
The PR has been reviewed by the code owners.

### Post-review checklist for the PR author

The PR author must check the checkboxes below before merging.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ]  The PR is ready for merge.
After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: .github/workflows/sync-upstream.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: sync-upstream
```
Comment (kenji-miyake) [inline .github/workflows/sync-upstream.yaml:1]: Please confirm your diff.

**Review by kenji-miyake — CHANGES_REQUESTED** [review]

File: .github/workflows/sync-upstream.yaml (hunk 1 above)
Comment (nobuotakamasa) [reply .github/workflows/sync-upstream.yaml:1]: Reverted unnecessary fixes.

Comment (kenji-miyake) [reply .github/workflows/sync-upstream.yaml:1]: Mm, there is no change. :pleading_face:

Comment (kenji-miyake) [reply .github/workflows/sync-upstream.yaml:1]: Please remove `sync-upstream.yaml`.

Comment (kenji-miyake) [reply .github/workflows/sync-upstream.yaml:1]: Also, please confirm the pull request guidelines.

**Comment by kenji-miyake** [comment]
@nobuotakamasa @ito-san Could you update the status of this PR? If it's not necessary anymore, please close it.

**Comment by ito-san** [comment]
> @nobuotakamasa @ito-san Could you update the status of this PR? If it's not necessary anymore, please close it.

There is a similar PR(https://github.com/autowarefoundation/autoware.universe/pull/1447) and I believe all of these PRs should be closed.

@nobuotakamasa 
May I close these PRs or would you please close it?

**Pull request closed without merging**